LICENSE

```diff
@@ -0,0 +1,94 @@
+Copyright 2019 Red Hat, Inc.,
```

Comment:
@richardfontana this should be file extension .md

Reply:
As far as I can tell there's no official markdown version of SIL OFL 1.1, otherwise I would do that.

Reply:
No file extension needed... I only added one in my pull request out of habit.